cmd/utils/flags.go

```diff
@@ -592,6 +592,39 @@ var (
 		Name:  "anchoring",
 		Usage: "Enable anchoring for service chain",
 	}
+	// KAS
+	ServiceChainKASAnchorFlag = cli.BoolFlag{
+		Name:  "kas.anchor",
```

Comment:
How about adding `sc` between `kas` and the flag name?
```suggestion
		Name:  "kas.sc.anchor",
```

Reply:
Reflected